client/dive-common/use/useModeManager.ts

```diff
@@ -224,7 +228,10 @@ export default function useModeManager({
 
   function handleUpdateRectBounds(frameNum: number, flickNum: number, bounds: RectBounds) {
     if (selectedTrackId.value !== null) {
-      const track = trackMap.get(selectedTrackId.value);
+      let track = trackMap.get(selectedTrackId.value);
+      if (selectedCamera.value !== 'default') {
+        track = camTrackMap[selectedCamera.value].get(selectedTrackId.value);
+      }
```

Comment:
I don't like having consumers responsible for all this logic, it's just more for bugs. 
 Possibly not part of this PR, but i think we need to 

1. Get rid of `trackMap` and replace it with `camTrackMap`
2. Replace everywhere that access the trackMap with a `getTrack(map, id, camera)` call that properly throws an error if either the camera or id don't exist.

Reply:
I've done this in the trackStore modification branch.  There really is no longer a trackMap, it's all `camMap` used in conjunction with different `getTrack` functions.  Except it's a little more complicated than a single `getTrack` function, because `getTrack(map, id, camera)` doesn't cover all needs, sometimes you just need the first trackId that's available and don't care what camera is selected, other times you need a conglomerate of all tracks across all cameras to display (in the case of the event Viewer for a selected track).  Finally you need a list of all available tracks to iterate over (For setting confidence pairs on all tracks regardless of camera, or setting track wide attributes)

Reply:
Got it, I remember talking about this now.  If we need some new functions or a more complicated `getTrack` that makes sense. I just don't want each function that needs to get a track to have to implement its own error handling / null checking.